documentation/proc-pages/physics-models/plasma_current/bootstrap_current.md

```diff
@@ -469,6 +467,231 @@ $$
 
 The model includes the toroidal diamagnetic current in the calculation due to the dataset, so `i_diamagnetic_current = 0` can only be used with it
 
+-------------------
+
+### ARIES Scaling | `bootstrap_fraction_aries()`
+
+Is selected by setting `i_bootstrap_current = 6`[^8]
+
+The source reference[^8] does not provide any info about the derivation of the formula. It is only stated like that shown below.
+
+$$
+a_1 = 1.10-1.165l_{\text{i}}+0.47l_{\text{i}}^2
+$$
+
+$$
+b_1 = 0.806-0.885l_{\text{i}}+0.297l_{\text{i}}^2
+$$
+
+$$
+C_{\text{BS}} = a_1+b_1\left(\frac{n(0)}{\langle n \rangle}\right)
+$$
+
+$$
+f_{\text{BS}} = C_{\text{BS}} \sqrt{\epsilon}\beta_{\text{p}}
+$$
+
+---------------------
+
+### Andrade Scaling 
+
+Is selected by setting `i_bootstrap_current = 7`[^9]
+
+Based off of 350 plasma profiles from Experimento Tokamak Esferico (ETE) spherical tokamak.
+Profiles were taken to be Gaussian shaped functions.
+
+The range of parameters from the discharges in ETE can be found in the table below:
+
+| Parameter             | Value     | Parameter             | Value     |
+|-----------------------|-----------|-----------------------|-----------|
+| $A$                   | 1.5       | $\alpha_{\text{T}_\text{e}}$ | 0.02      |
+| $R_0$                 | 0.3 $\text{[m]}$     | $\alpha_{\text{T}_\text{i}}$ | 2         |
+| $p(0)$                | 15 $\text{[kPa]}$    | $\kappa(a)$           | 2         |
+| $T_{\text{e,i}}(0)$   | 1 $\text{[keV]}$     | $\delta$              | 0.3       |
+| $T_{\text{e,i}}(a)$   | 0.1 $\text{[keV]}$   | $I_{\text{p}}$        | 200 $\text{[kA]}$    |
+| $\alpha_{\text{p}}$   | 3         | $B_0$                 | 0.4 $\text{[T]}$     |
+| $\beta$               | 4-10%     | $Z_{\text{eff}}$      | 1         |
```

Comment:
Should you add what the parameters actually are? If so it probably doesn't make sense to continue with the doubled up table

Reply:
They are meant to be the standard physics variables, I dont see the need to re-state for every new model what the standard symbols represent